charts/timescaledb-single/values.yaml

```diff
@@ -23,6 +23,11 @@ image:
   tag: pg14.3-ts2.7.0-p0
   pullPolicy: Always
 
+## @param priorityClassName Name of the existing priority class to be used by kafka pods
+## Ref: https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/
+##
+priorityClassName: "timescaledb"
```

Comment:
Could you make it empty by default? Otherwise default installation may fail when the priority class does not exist.